admin/views/class-yoast-feature-toggle.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class representing a feature toggle.
+ */
+class Yoast_Feature_Toggle {
+
+	/** @var string Feature toggle identifier. */
+	protected $name = '';
+
+	/** @var string Name of the setting the feature toggle is associated with. */
+	protected $setting = '';
+
+	/** @var string Feature toggle label. */
+	protected $label = '';
+
+	/** @var array Additional feature arguments. */
+	protected $args = array();
```

Comment:
`$args` is a little bit vague. I think we should give it a more descriptive name.